plugins/javadoc/src/main/kotlin/javadoc/renderer/JavadocContentToTemplateMapTranslator.kt

```diff
@@ -73,45 +71,49 @@ internal class JavadocContentToTemplateMapTranslator(
             )
         }
 
-        internal fun templateMapForPackagePageNode(node: JavadocPackagePageNode): TemplateMap {
+        fun templateMapForPackagePageNode(node: JavadocPackagePageNode): TemplateMap {
             return mapOf(
                 "kind" to "package"
             ) + templateMapForJavadocContentNode(node.content)
         }
 
-        internal fun templateMapForFunctionNode(node: JavadocFunctionNode): TemplateMap {
-            val (modifiers, signature) = node.modifiersAndSignature
+        fun templateMapForFunctionNode(node: JavadocFunctionNode): TemplateMap {
             return mapOf(
-                "signature" to htmlForContentNode(node.signature, contextNode),
                 "brief" to htmlForContentNodes(node.brief, contextNode),
                 "parameters" to node.parameters.map { templateMapForParameterNode(it) },
-                "inlineParameters" to node.parameters.joinToString { "${it.type} ${it.name}" },
-                "modifiers" to htmlForContentNode(modifiers, contextNode),
-                "signatureWithoutModifiers" to htmlForContentNode(signature, contextNode),
+                "inlineParameters" to node.parameters.joinToString { "${htmlForContentNode(it.type, contextNode)} ${it.name}" },
```

Comment:
I'd personally remove the string interpolation for the function call here as it looks ugly

Reply:
done